src/strand-view-manager/strand-view-manager.js

```diff
@@ -0,0 +1,79 @@
+/**
+ * @license
+ * Copyright (c) 2016 MediaMath Inc. All rights reserved.
+ * This code may only be used under the BSD style license found at http://mediamath.github.io/strand/LICENSE.txt
+
+*/
+(function (scope) {
+
+	scope.ViewManager = Polymer({
+		is: "strand-view-manager",
+
+		behaviors: [
+			StrandTraits.Refable
+		],
+
+		properties: {
+
+			index: {
+				type:Number,
+				value: 0,
+				notify:true,
+				observer:'_indexChanged'
+			},
+			views:{
+				type:Array,
+				value: function() {
+					return [];
+				},
+				notify:true,
+				observer:'_viewsChanged'
+			}
+		},
+
+		listeners:{
+			'dom-change':'_domChanged'
+		},
+
+		_indexChanged: function(index, old) {
+			this.set('views.'+old+".selected",false);
+			this.set('views.'+index+".selected",true);
+		},
+
+		_viewsChanged: function() {
+			var sel = this.views.some(function(v) {
+				return v.selected;
+			});
+			if (!sel) {
+				this.set('views.0.selected',true);
+			}
+		},
+
+		_domChanged: function(e) {
+			var name = this.get('views.' + this.index + '.name');
+			if (e.target && e.target.id.indexOf('view-'+ name) !== -1) {
```

Comment:
bad refactoring. originally it was just checking for view- but in the tests i figured out really quickly that was basically completely useless for the 'end' developer that might want to use that event to trigger actions in their views (ie send data to a grid)